libwasmvm/Cargo.toml

```diff
@@ -30,8 +30,8 @@ default = []
 backtraces = []
 
 [dependencies]
-cosmwasm-std = { git = "https://github.com/line/cosmwasm", rev = "811d14d8f36dc807c0f8cdf6c36d6ca1d09377b4", features = ["iterator","staking","stargate"] }
-cosmwasm-vm = {  git = "https://github.com/line/cosmwasm", rev = "811d14d8f36dc807c0f8cdf6c36d6ca1d09377b4", features = ["iterator","staking","stargate"] }
+cosmwasm-std = { git = "https://github.com/line/cosmwasm", version = "0.16.3-0.5.1", features = ["iterator","staking","stargate"] }
```

Comment:
in the previous version, we used the 'tag'
# cosmwasm-std = { git = "https://github.com/line/cosmwasm", tag = "v0.14.0-0.4.0", features = ["iterator","staking","stargate"] }
# cosmwasm-vm = {  git = "https://github.com/line/cosmwasm", tag = "v0.14.0-0.4.0", features = ["iterator","staking","stargate"] }

Reply:
Nice finding! Thanks!